package.json

```diff
@@ -13,38 +13,71 @@
     "src",
     "dist"
   ],
-  "main": "src",
+  "main": "dist/ldclient.cjs.js",
+  "module": "dist/@ldclient.es.js",
+  "browser": "dist/@ldclient.umd.js",
   "scripts": {
-    "build-lib": "webpack src/index.js dist/ldclient.js",
-    "build-min": "webpack -p src/index.js dist/ldclient.min.js",
-    "build": "node ./scripts/build.js",
-    "watch": "webpack --watch src/index.js dist/ldclient.js",
-    "test": "karma start --single-run",
-    "test:watch": "karma start --auto-watch --reporters=dots",
-    "prepublish": "node ./scripts/build.js"
+    "preversion": "npm run build",
+    "prebuild": "rimraf dist/**",
+    "lint": "eslint --format 'node_modules/eslint-formatter-pretty' src",
+    "format": "npm run format:md && npm run format:js",
+    "format:md": "prettier --parser markdown --write '*.md'",
+    "format:js": "prettier --write 'src/**/*.js'",
+    "build": "cross-env npm run build:umd && npm run build:umd:min && npm run build:es && npm run build:cjs",
+    "build:watch": "cross-env npm run build:es -- -w",
+    "build:umd": "cross-env NODE_ENV=development rollup -c rollup.config.js -f umd -i src/index.js -o dist/ldclient.umd.js",
+    "build:umd:min": "cross-env NODE_ENV=production rollup -c rollup.config.js -f umd -i src/index.js -o dist/ldclient.umd.min.js",
+    "build:es": "cross-env NODE_ENV=development rollup -c rollup.config.js -f es -i src/index.js -o dist/ldclient.es.js",
+    "build:cjs": "cross-env NODE_ENV=development rollup -c rollup.config.js -f cjs -i src/index.js -o dist/ldclient.cjs.js",
+    "test": "karma start --log-level debug",
+    "test:watch": "karma start --auto-watch --reporters=dots"
   },
   "types": "./src/index.d.ts",
   "devDependencies": {
-    "chai": "3.5.0",
-    "gzip-size": "3.0.0",
-    "in-publish": "2.0.0",
-    "karma": "0.13.22",
+    "babel-core": "^6.26.0",
+    "babel-eslint": "^8.0.2",
+    "babel-plugin-external-helpers": "^6.22.0",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-react": "6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
+    "chai": "4.1.2",
+    "cross-env": "^5.1.1",
+    "karma": "1.7.1",
     "karma-chai": "0.1.0",
-    "karma-chrome-launcher": "1.0.1",
-    "karma-mocha": "1.0.1",
-    "karma-mocha-reporter": "2.0.4",
-    "karma-phantomjs-launcher": "0.2.1",
-    "karma-phantomjs-shim": "1.1.2",
+    "karma-chrome-launcher": "2.2.0",
```

Comment:
I bumped most of the karma dependencies. Maybe another nice thing would be to enable Greenkeeper on the repository so dependencies don't get super outdated.